x-pack/plugins/observability_solution/apm/server/routes/alerts/rule_types/anomaly/register_anomaly_rule_type.ts

```diff
@@ -70,296 +90,302 @@ export function registerAnomalyRuleType({
   ml,
   ruleDataClient,
 }: RegisterRuleDependencies) {
-  const createLifecycleRuleType = createLifecycleRuleTypeFactory({
-    logger,
-    ruleDataClient,
-  });
+  if (!alerting) {
+    throw new Error(
+      'Cannot register anomaly rule type. Both the actions and alerting plugins need to be enabled.'
```

Comment:
Is the actions plugin actually required?  Can't tell from this PR anyway (or I didn't notice).

Reply:
You are right I don't think it's actually required, thanks for catching! I will update the error message in this PR and the others.

Reply:
Resolved in this commit [193c433](https://github.com/elastic/kibana/pull/179196/commits/193c433b4a63856a0af2fd69d5dadfb519f7dd40)